go.mod

```diff
@@ -0,0 +1,5 @@
+module github.com/honeycombio/dynsampler-go
+
+go 1.12
+
+require github.com/stretchr/testify v1.6.1
```

Comment:
Noting here for posterity: this is the version that still works with Go 1.12. testify 1.7 requires Go 1.13.